src/psyclone/tests/psyir/transformations/intrinsics/dotproduct2code_trans_test.py

```diff
@@ -37,10 +37,12 @@
 transformation.
 
 '''
+# pylint: disable=protected-access
```

Comment:
Please remove this two lines.

Reply:
Removed. I had to add too-many-arguments to the .pylintrc file. I also found a couple of other pylint errors which I fixed.